src/OpenTelemetry.Exporter.Geneva/MsgPackExporter/MessagePackSerializer.cs

```diff
@@ -563,6 +563,8 @@ public static int Serialize(byte[] buffer, int cursor, object obj)
                 return SerializeArray(buffer, cursor, v);
             case DateTime v:
                 return SerializeUtcDateTime(buffer, cursor, v.ToUniversalTime());
+            case DateTimeOffset v:
```

Comment:
Do we want to include `TimeSpan` as well?

Reply:
I guess not now.